react/features/toolbox/components/index.js

```diff
@@ -3,4 +3,5 @@
 export * from './_';
 export { default as AudioMuteButton } from './AudioMuteButton';
 export { default as HangupButton } from './HangupButton';
+export { default as Notice } from './Notice';
```

Comment:
This will break on mobile. But come to think of it, why is this notice in the toolbox, if it then gets rendered int he Conference? I think it belongs in features/conference.

Reply:
I must admit I'm a bit of lost here. I added it because it is where it  was in the original commit 59a74153dc9ec0432cbc6f1fac5ef9c1b3566cd4. 

Any kind of guide would be well received by me.

Reply:
Hi Pablo. Sorry, I realized I was waaaay too terse there. This file gets imported on all platforms (web and mobile) and when Notice.js is imported on mobile it will give an error because it contains DOM elements, which mobile doesn't use because we use React Native there.

In order to avoid this you can just rename the file to Notice.web.js and create an empty Notice.naive.js.

While reading through the PR though, I noticed (pun intended) that it doesn't make much sense (conceptually) to have the Notice component in the toolbox feature if it's then rendered just in the Conference, where it rightfully belongs. I know it might have been there before, but the toolbox has been rewritten a number of times (this time hopefully for good!) so the abstractions are clearer now.

That's why I'd like to see the Notice component in the conference feature instead of having it in the Conference feature.

Again, sorry for being too terse, I certainly value your contribution to the project. ❤️

Reply:
Don't worry, I only needed context to understand the changes. Now, thanks a lot for the detailed explanation.

I just send a additional patch (c1af213) fixing the code based on your comments.  Let me know if you have more comments or suggestions.

Once you are agree, I'll squash the patches in and one single commit ready to be merged.

Very happy to help.